recipe/meta.yaml

```diff
@@ -19,11 +20,12 @@ build:
 requirements:
   host:
     - python
-    - flit-core >=3.2,<4
     - pip
-    - wheel
+    - hatchling
+    - hatch-nodejs-version
   run:
     - python
+    - importlib_metadata >=3.6
```

Comment:
You could set this instead: `- importlib_metadata >=3.6. # [py<38]` as this only applied to python lower than 3.8.
https://github.com/jupyter/nbformat/blob/5.7.0/pyproject.toml#L29

Reply:
Okay, thanks!